packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -72,12 +82,31 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
   scope.childScopes.forEach(findOfficeApiReferencesInScope);
 }
 
+function isMethodFunction(node: TSESTree.MemberExpression): boolean {
```

Comment:
I know I put this in my own comment, but reading it here make me realize "MethodFunction" is redundant :-p . . . it should just be "isFunction" or "isMethod"

Also . . . I think you still need the 'findTopLevelExpression' . . . otherwise this won't work for thing like Ref.prop.prop.method()